lighthouse-cli/test/smokehouse/seo/expectations.js

```diff
@@ -185,8 +185,8 @@ module.exports = [
     audits: {
       'tap-targets': {
         score: (() => {
-          const PASSING_TAP_TARGETS = 11;
-          const TOTAL_TAP_TARGETS = 12;
+          const PASSING_TAP_TARGETS = 12;
+          const TOTAL_TAP_TARGETS = 13;
```

Comment:
I'd would be nice to assert each tap target from the artifact here, maybe we can modify the expectations logic to allow that?